src/_data/LiveData.js

```diff
@@ -56,15 +56,14 @@ export const changeLanguage = langCode => {
 };
 
 const initUnauthorized = async () => {
-
-};
-
-const initAuthorized = async (authData, store) => {
     api.getActiveSymbolsFull();
     api.getTradingTimes(new Date());
     api.getAssetIndex();
     api.getServerTime();
     api.getCandlesForLastNDays('R_100', 30);
```

Comment:
Yes, those dependencies cause the sign in to fail, because the redirect happens before data is ready.

since ws is async, does making these api before hand make a big difference?

I'll revert this part though